docs/components/modeler/bpmn/compensation-handler/compensation-handler.md

```diff
@@ -0,0 +1,79 @@
+---
+id: compensation-handler
+title: "Compensation"
+description: "A compensation handler is an activity used to undo tasks that have already been executed."
+---
+
+Activities that are associated to a compensation boundary event have a compensation marker. These activities are called
+**compensation handlers** and are in charge of reverting the effects of the activity with the compensation boundary
+event, the compensation activity.
+
+![Compensation marker example](assets/compensation-marker-example.png)
+
+When the process instance reaches the compensation throw event, it invokes the compensation handler `undo A` if the
+task `A` is completed before. If the task has been completed more than once, the compensation handler is invoked for the
+same amount.
+
+Read more about triggering the compensation in the [compensation events documentation](../compensation-events/compensation-events.md).
+
+## Embedded subprocess as compensation handler
+
+![Process with subprocess as compensation handler](assets/subprocess-compensation-handler.png)
+
+The subprocess contains the steps to undo the actions of the compensation activity. Using a subprocess can be useful if
+a sequence of steps is required to undo the actions of the activity.
+
+## Call activity as compensation handler
+
+![Process with call activity as compensation handler](assets/call-activity-compensation-handler.png)
+
+The call activity contains the steps to undo the actions of the compensation activity. Using a call activity can be
```

Comment:
Do we mean "Using a call activity can be
useful in combination with a compensation activity since compensation isn't propagated to the child
process."?

Reply:
Yes. Good hint. :+1: I rephrased the sentence.